RevenueCatUI/CustomerCenter/Views/AppUpdateWarningView.swift

```diff
@@ -40,30 +40,47 @@ struct AppUpdateWarningView: View {
     }
 
     var body: some View {
-        List {
-            Section {
-                CompatibilityContentUnavailableView(
-                    localization[.updateWarningTitle],
-                    systemImage: "arrow.up.circle.fill",
-                    description: Text(localization[.updateWarningDescription])
-                )
+        Color(colorScheme == .light ? UIColor.secondarySystemBackground : UIColor.systemBackground)
```

Comment:
Can this color be extracted since it's used in 2 different places? Also, I wonder if we should just default to default colors, but not sure how that would look like. I'm ok with this change!

Reply:
I tried not doing this but couldn't find a combination that looked like the previous version. 🤔

Reply:
Yeah, I also thought about this. But note that the two places where it's used the colors are inverted. So it's not exactly the same.

My first though was that this felt unnecessary and that it should suffice using `UIColor.secondarySystemBackground` in one place and `UIColor.systemBackground` in the other one.

But then I checked the `Settings` app and it turns out that it uses the exact same combination of colors in light and dark mode, so this is the right way to go, IMO.